CommandLine/libEGM/filesystem.cpp

```diff
@@ -0,0 +1,58 @@
+#include "filesystem.h"

```

Comment:
You can inline this file using templates and/or `using`, but it should be fine like this.

Reply:
I don't know what you mean.